packages/marathon/build

```diff
@@ -19,6 +19,8 @@ StartLimitInterval=0
 RestartSec=15
 PermissionsStartOnly=True
 EnvironmentFile=/opt/mesosphere/environment
+EnvironmentFile=-/opt/mesosphere/etc/marathon
```

Comment:
Might want to double-check this. This didn't work for all options. I could also be remembering Metronome.